app/controllers/concerns/account_scope_filtering.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module AccountScopeFiltering
+
+  private
+
+  def apply_account_filters(scope, filter_params)
+    return scope unless SettingsHelper.feature_on?(:account_tabs)
```

Comment:
Wouldn't filter_params already be an empty hash if this FF is on? is this FF check needed?

Reply:
It's needed because there's an `else` clause that would apply `not_suspended` scope